demos/dashboards-investor-preferences/demo.js

```diff
@@ -0,0 +1,413 @@
+                                        'op': 'eq',
+                                        'value': 'Y'
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    ];
+
+    const columns = [];
+    for (const key in headerFormats) {
+        columns.push({
+            id: `InvestorPreferences_${key}`,
+            header: {
+                format: headerFormats[key] || key
+            },
+            cells: {
+                formatter () {
+                    if (this.column.id === 'InvestorPreferences_StarRatingM255') {
+                        return Array(this.value).fill(starIcon).join('');
+                    }
+                    if (typeof this.value === 'boolean') {
+                        return this.value ? '✅' : '❌';
+                    }
+                    return this.value ?? '';
+                }
+            }
+        });
+    }
+
+    const board = Dashboards.board('container', {
+        dataPool: {
+            connectors: [
+                {
+                    id: 'investor-preferences',
+                    type: 'MorningstarInvestorPreferences',
+                    options: {
+                        page: 1,
+                        pageSize: 20,
+                        languageId: 'en-GB',
+                        currencyId: 'USD',
+                        securityDataPoints: secIds,
+                        universeIds: ['FOGBR$$ALL'],
+                        calculatedDataPoints,
+                        filters: [{
+                            dataPointId: 'UserPref0',
+                            comparatorCode: 'EQ',
+                            value: 'True'
+                        }],
+                        postman: {
+                            environmentJSON: postmanJSON
+                        }
+                    }
+                }
+            ]
+        },
+        components: [
+            {
+                renderTo: 'dashboard-col-1',
+                connector: {
+                    id: 'investor-preferences'
+                },
+                type: 'DataGrid',
+
+                dataGridOptions: {
+                    editable: false,
+                    columns
+                },
+                title: 'Investor Preferences'
+            }
+        ]
+    });
+
+    board.dataPool.getConnector('investor-preferences').then(connector => {
+        loadingLabel.style.display = 'none';
+        document.getElementById('total').innerHTML =
+            `total - ${connector.metadata.total}`;
+        document.getElementById('page').innerHTML =
+            `page - ${connector.metadata.page}`;
+        document.getElementById('total-pages').innerHTML =
+            `out of ${Math.ceil(connector.metadata.total / connector.metadata.pageSize)}`;
+    });
+
+
+    /**
+     * Set prev/next page
```

Comment:
Changing pages reorders columns, could we prevent that from happening? 

![image](https://github.com/user-attachments/assets/38b59d0c-a2a8-4471-ab8e-4ff95074fef4)

Reply:
The height also changes - it shouldn't